build_tools/ci/cpu_comparison/run.py

```diff
@@ -467,9 +440,50 @@ def _execute(self, config):
         matmul_template_dir = config.file_dir / "matmul_template"
         template_name = matmul_template_dir / "matmul_bias_MxK_KxN_N.mlir"
         self.generate(config, template_name)
+        self.add_aie_compilation_flags(
+            [
+                "--iree-amdaie-matmul-elementwise-fusion",
+                "--iree-amdaie-num-rows=2",
```

Comment:
Doesn't this change the default for all matmuls from 4x4 to 2x2? Maybe not what we want? 

I'd be interested to know how many of the tests 
```
for n_rows in [1,4]:
  for n_cols in [1,2,3,4]:
    # run matmul with these values  (on npu1_4col for say M=N=K=3*256).
```
work

Reply:
It being shown in a confusion way here, but these changes are actually in `MatmulThinBias`, not `Matmul`

Reply:
Yes, it's added to `MatmulThinBias` class and used for matmul-elementwise tests in AIR.

Reply:
Ha, I totally missed that... 

![{8C56B1BC-EB69-4D62-9344-F07F07905261}](https://github.com/user-attachments/assets/e5440e60-81f2-4ecf-bcdc-751e362986c6)